### PR TITLE
feat(service): add proxy-http socks

### PR DESCRIPTION
**Pull request by 13ilya**
https://github.com/firewalld/firewalld/issues/1389

### CODE REVIEW

**Comment by IlyaIndigo** [comment]
How do you pass Semantic PR?

**Comment by erig0** [comment]
The commit message (first line) must match one of the keywords [specified in the config](https://github.com/firewalld/firewalld/blob/main/.github/semantic.yml).

e.g. `feat(service): add http proxy`

**Comment by 13ilya** [comment]
Can you please tell me what is wrong with my PR?
Why is it being ignored and not checked?

File: config/services/proxy.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>proxy</short>
+  <description>HTTP/HTTPS proxy server</description>
+  <port protocol="tcp" port="3128"/>
```
Comment (erig0) [inline config/services/proxy.xml:5]: Port `3128` belongs to `ndl-aas` according to ICANN. Furthermore the service name `proxy` is too generic.

Where are you getting port `3128` ?

**Review by erig0 — CHANGES_REQUESTED** [review]

File: config/services/proxy.xml (hunk 1 above)
Comment (13ilya) [reply config/services/proxy.xml:5]: 1 Port 3128 is used by default by all proxy servers I know. 
2 I suggested other options in #1389, such as proxy-http and proxy-socks, and I'm fine with those too.
3 https://github.com/firewalld/firewalld/blob/main/config/services/squid.xml This in no way prevented you from adding the squid service with an even worse name, even though squid is far from being the only proxy server.

Comment (erig0) [reply config/services/proxy.xml:5]: I'd like to see it renamed to `proxy-http`.

Comment (erig0) [reply config/services/proxy.xml:5]: `socks` is okay. That's the IANA assigned name.

**Review by erig0 — APPROVED** [review]
I split your commit into two. I'll merge after CI.

**Pull request merged**